proj/cmake/platform_macosx.cmake

```diff
@@ -28,6 +28,15 @@ list( APPEND SRC_SET_AUDIO_COCOA
 	${CINDER_SRC_DIR}/cinder/audio/cocoa/FileCoreAudio.cpp
 )
 
+list( APPEND SRC_SET_QTIME
+	${CINDER_SRC_DIR}/cinder/qtime/AvfUtils.mm
+	${CINDER_SRC_DIR}/cinder/qtime/AvfWriter.mm
+	${CINDER_SRC_DIR}/cinder/qtime/MovieWriter.cpp
+	${CINDER_SRC_DIR}/cinder/qtime/QuickTimeGlImplAvf.cpp
```

Comment:
I think the OBJCPP files are not built in themselves. This list is used to change their build flags only here https://github.com/cinder/Cinder/blob/android_linux/proj/cmake/platform_macosx.cmake#L59
So I believe adding them twice is necessary at the moment.